cfme/tests/cloud/test_providers.py

```diff
@@ -1215,52 +1215,51 @@ def test_refresh_with_stack_without_parameters(provider, request, stack_without_
     provider.validate_stats(ui=True)
 
 
-@test_requirements.ec2
-@pytest.mark.manual
-def test_ec2_public_images():
+@test_requirements.cloud
+@pytest.mark.long_running
+@pytest.mark.provider([AzureProvider, EC2Provider], scope="function")
+def test_public_images_enable_disable(setup_provider, request, appliance, provider):
     """
+    Bugzilla:
+        1612086
+        1491330
+    The easiest way to simulate AWS API Limit for > 200 items is to enable
+    and disable public images.
+    So test for testing public images and for testing AWS API Limit is combined in this test.
     Polarion:
         assignee: mmojzis
         caseimportance: critical
-        initialEstimate: 2/3h
+        initialEstimate: 1 1/2h
```

Comment:
1.5 hrs for this test?

Reply:
Yes, it sometimes take even more.

Reply:
This initialestimate is for how long it takes to manually execute the test.  How long does it take to actually execute?

Current PRT result has failed:
```
File "/cfme/cfme_tests/cfme/cloud/instance/__init__.py", line 12, in <module>
    from cfme.common import BaseLoggedInPage
ImportError: Error importing plugin "cfme.markers.env": cannot import name 'BaseLoggedInPage' from 'cfme.common' (/cfme/cfme_tests/cfme/common/__init__.py)```

Reply:
It takes about 1.5 hours to execute it.